sea-orm-macros/src/derives/active_enum.rs

```diff
@@ -101,6 +101,8 @@ impl ActiveEnum {
                     } else if meta.path.is_ident("num_value") {
                         is_int = true;
                         num_value = Some(meta.value()?.parse::<LitInt>()?);
+                    } else if meta.path.is_ident("display_value") {
+                        Some(meta.value()?.parse::<LitStr>()?);
```

Comment:
I believe this is not needed?

Reply:
Since the code for DeriveActiveEnum will panic if there is some unrecognised attributes, I added this so that deriving both Display and Active Enum would not make the code panic.